src/main/java/chess/model/piece/PieceText.java

```diff
@@ -0,0 +1,54 @@
+package chess.model.piece;
+
+import java.util.Arrays;
+import java.util.function.Predicate;
+
+public enum PieceText {
+    WHITE_PAWN("p", piece -> piece instanceof Pawn && piece.isSameSide(Side.WHITE), new WhitePawn()),
```

Comment:
```suggestion
    WHITE_PAWN(
        "p",
        piece -> piece instanceof Pawn && piece.isSameSide(Side.WHITE),
        new WhitePawn()
    ),
```
줄마꿈을 해서 가독성을 높여보는건 어떨까요?